x-pack/plugins/security/server/authentication/api_keys.ts

```diff
@@ -188,7 +189,7 @@ export class APIKeys {
    * Tries to grant an API key for the current user.
    * @param request Request instance.
    */
-  async grantAsInternalUser(request: KibanaRequest) {
+  async grantAsInternalUser(request: KibanaRequest, createParams: CreateAPIKeyParams) {
```

Comment:
__optional nit:__ missing JSDoc for `createParams` (I admit these JSDocs are useless most of the time, so feel free to ignore to not wait for CI run once again).

Reply:
I am going to merge w/o the JSDoc comment, just in the interest of time given our tight schedule today. I'll try to be more diligent about this going forward though!